react-native/ios/RealmReact/RealmReact.mm

```diff
@@ -69,7 +71,7 @@ - (JSGlobalContextRef)ctx;
         object_setIvar(executor, contextIvar, rctJSContext);
     }
 
-    return [rctJSContext ctx];
+    return [rctJSContext context].JSGlobalContextRef;
```

Comment:
It does. Looks like it has been around since 0.14.0. We should have been using it earlier but its addition was unnoticed.